sdk/search/Azure.Search.Documents/src/Generated/Models/AnalyzeRequest.cs

```diff
@@ -41,7 +41,7 @@ internal AnalyzeRequest(string text, AnalyzerName? analyzer, TokenizerName? toke
         }
 
         /// <summary> The text to break into tokens. </summary>
-        public string Text { get; }
+        public string Text { get; set; }
```

Comment:
From @KrzysztofCwalina, required properties should only be settable from within the constructor.

Reply:
Same applies below.